pkg/antctl/raw/multicluster/deploy/deploy_helper.go

```diff
@@ -121,7 +121,21 @@ func createResources(cmd *cobra.Command, apiGroupResources []*restmapper.APIGrou
 			if !kerrors.IsAlreadyExists(err) {
 				return err
 			}
-			fmt.Fprintf(cmd.OutOrStdout(), "%s/%s already exists\n", unstructuredObj.GetKind(), unstructuredObj.GetName())
+			existingRes, err := dri.Get(context.TODO(), unstructuredObj.GetName(), metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			existingVersion := existingRes.GetResourceVersion()
+			unstructuredObj.SetResourceVersion(existingVersion)
+			var updatedObj *unstructured.Unstructured
+			if updatedObj, err = dri.Update(context.TODO(), unstructuredObj, metav1.UpdateOptions{}); err != nil {
+				return err
+			}
+			if updatedObj.GetResourceVersion() != existingVersion {
+				fmt.Fprintf(cmd.OutOrStdout(), "%s/%s configured\n", unstructuredObj.GetKind(), unstructuredObj.GetName())
```

Comment:
configured -> updated?

Reply:
When using `kubectl apply` with a updated manifests, it will show `configured`, maybe we should keep them the same?

Reply:
No strong opinion. Updated sounds better to me, but up to you.